src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/RedirectHandler.cs

```diff
@@ -33,14 +33,22 @@ public RedirectHandler(int maxAutomaticRedirections, HttpMessageHandler initialI
             Uri redirectUri;
             while ((redirectUri = GetUriForRedirect(request.RequestUri, response)) != null)
             {
-                response.Dispose();
-
                 redirectCount++;
+
                 if (redirectCount > _maxAutomaticRedirections)
                 {
-                    throw new HttpRequestException(SR.net_http_max_redirects);
```

Comment:
Delete the string from resx or there is some code still using it?

Reply:
I'll remove the string. There are no other uses of it.